### PR TITLE
Add false-positives documentation

### PR DESCRIPTION
**Pull request by MVrachev**
Closes: https://github.com/vmware/precaution/issues/118

I was thinking that it would be a good idea to give information for the users how to handle false positives.

On another hand that means that non secure code may be introduced into the main master branch because the pull reuqest author may use this feature incorrectly.

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

### CODE REVIEW

File: docs/false-pos.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+# False positives and how to handle them
+
+No tool is perfect, neither is Precaution and false positives may happened.
```
Comment (evqna) [inline docs/false-pos.md]:
```suggestion
As with all automated detection tools there will be cases of false positives.
```

File: docs/false-pos.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+The way to handle this situation is to write #nosec in your code
```
Comment (evqna) [inline docs/false-pos.md]:
```suggestion
If Precaution reports a failure that has been manually verified as being safe, it is possible to annotate the code with a `#nosec` comment.
```

File: docs/false-pos.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Python example
+
+Although this line may cause a potential security issue, it will not be reported:
+
+``` self.process = subprocess.Popen('/bin/echo', shell=True)  # nosec ```
+
+## Go example
+
+Same here, with #nosec comment you basically say: "I know what I am doing don't warn me":
```
Comment (evqna) [inline docs/false-pos.md]:
```suggestion
In Go the annotation has to be inside a line or block comment:
```

**Comment by MVrachev** [comment]
That would be awesome think to have Eric, but still I am not sure how we would do this. :(
But still I will create an issue about this.

**Pull request merged**